### PR TITLE
Added Documentation for using Tailwind with Gatsby

### PR DESCRIPTION
**Pull request by jakedohm**
## Description

I added a document with a walkthrough of how to get Tailwind CSS up and running with Gatsby via PostCSS. Based on: https://dev.to/jakedohm_34/using-tailwind-with-gatsby-js-10fj

Let me know if anything is missing, or any corrections need to be made!

### CODE REVIEW

File: docs/docs/tailwind-css.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+You can now use the `@tailwind` directives to add Tailwind's utilites, preflight, and components into your CSS. You can also use `@apply` and all of Tailwind's other directives and functions!
+
+To learn more about how to use Tailwind in your CSS, visit the [Tailwind Documentation](https://tailwindcss.com/docs/installation#3-use-tailwind-in-your-css)
+
+
+### Other resources
+
+- [Introduction to postcss](https://www.smashingmagazine.com/2015/12/introduction-to-postcss/)
+- [Tailwind Documentation](https://tailwindcss.com/)
```
Comment (LekoArts) [inline docs/docs/tailwind-css.md]: Linking to starters could be handy!

```suggestion
- [Tailwind Documentation](https://tailwindcss.com/)
- [Gatsby starters that use Tailwind](https://www.gatsbyjs.org/starters/?c=Styling%3ATailwind&v=2)
```

**Comment by hagnerd** [comment]
Also food for thought, I would recommend adding in how to use it with PurgeCSS since Tailwind generates so much CSS and the end user likely will only use a small portion of what was generated.

**Comment by jakedohm** [comment]
I think I'm going to re-structure the article like so:

Tailwind Installation & Configuration
- Installing Tailwind
- Adding config file

Using Tailwind
- PostCSS
    - Installation/Usage steps
- CSS-in-JS
    - Installation/Usage steps

Since the you have to install and configure Tailwind for _both_ methods, I think it makes sense to only do that stuff in one place, and *then* show them how to implement either method. Does that sound good to you @LekoArts?

@hagnerd I'll also add instructions for how to enable PurgeCSS!

**Comment by LekoArts** [comment]
Yeah, sounds good!

**Comment by calcsam** [comment]
Hey @jakedohm, wanted to check where you were on this? More and more people are asking about Gatsby and Tailwind so would be great to get this in if you're able to finish it up!

**Comment by jakedohm** [comment]
@calcsam @LekoArts hi again! I've updated the docs in my latest commit to show steps for usage with both PostCSS (standard) and CSS-in-JS.

I think this should be pretty close, but feedback is welcome!

**Comment by calcsam** [comment]
This looks great, thanks @jakedohm! I'll add a follow-up PR adding this to the docs hierarchy!

**Pull request merged**